packages/aws-cdk/lib/api/aws-auth/awscli-compatible.ts

```diff
@@ -196,4 +193,26 @@ function readIfPossible(filename: string): string | undefined {
     debug(e);
     return undefined;
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Ask user for MFA token for given serial
+ *
+ * Result is send to callback function for SDK to authorize the request
+ */
+async function tokenCodeFn(serial: string, cb: (err?: Error, token?: string) => void): Promise<void> {
+  debug('Require MFA token for serial', serial);
+  try {
+    const { token }: { token: string} = await inquirer.prompt({
+      name: 'token',
+      type: 'input',
+      default: '',
+      message: `MFA token for ${serial}`
+    });
+    debug('Successfully got MFA token:', token);
```

Comment:
do we want to log the token even though it's short lived? 
There's an outstanding story to start logging all CLI output for auditing purposes.